ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -570,7 +563,7 @@ export const ConceptHomepage = fp.flow(
                           browseSurvey={() =>
                             this.browseDomain(Domain.SURVEY, survey.name)
                           }
-                          updating={surveysLoading.includes(survey.name)}
+                          updating={loadingConceptCounts}
```

Comment:
Same as above, can remove `updating` and spinner from the `SurveyCard` component